rad/src/lib.rs

```diff
@@ -126,7 +126,16 @@ pub async fn run_retrieval_report(
             let result =
                 run_retrieval_with_data_report(retrieve, response_string, context, settings);
 
-            log::debug!("Result for URL {}: {:?}", retrieve.url, result);
+            match &result {
+                Ok(report) => {
+                    log::debug!(
+                        "Successful result for source {}: {:?}",
+                        retrieve.url,
+                        report
```

Comment:
This change got lost somewhere?

```suggestion
                        report.as_ref().map(|report| report.result)
```